src/js/components/account/change-password.js

```diff
@@ -0,0 +1,92 @@
+import { API } from "../../api";
+import {
+  getDiv,
+  getForm,
+  getInput,
+  getLabel,
+  getSubmitButton,
+} from "../../utils/element-factory";
+
+export class ChangePassword {
+  constructor() {
+    const $successTemplate = $(".styles .form-styles .w-form-done").clone();
+    const $failTemplate = $(".styles .form-styles .w-form-fail").clone();
```

Comment:
With the new webflow extract merged, you will need to update this to the following:

```suggestion
    const $successTemplate = $(".components .form-styles .w-form-done").clone();
    const $failTemplate = $(".components .form-styles .w-form-fail").clone();
```

Reply:
Thanks @schalkneethling 👍🏼